absl/synchronization/internal/create_thread_identity.cc

```diff
@@ -67,6 +67,33 @@ static intptr_t RoundUp(intptr_t addr, intptr_t align) {
   return (addr + align - 1) & ~(align - 1);
 }
 
+namespace {
+
+  void ResetThreadIdetity(base_internal::ThreadIdentity* identity) {
```

Comment:
https://github.com/abseil/abseil-cpp/blob/master/CONTRIBUTING.md#coding-style

Please be sure to follow our style-guide, or use `clang-format` to automatically format your code. In particular, we don't intend here <https://google.github.io/styleguide/cppguide.html#Namespaces>.

However, local style within this file uses `static` instead of the anonymous namespace for functions that aren't exported from a file, so prefer that.

Reply:
Changed to `static void ResetThreadIdentity`.